From d319dc8713c7db1eb54d0474c7c87aeaf1064b2f Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Thu, 24 Jan 2013 16:24:15 +0100
Subject: CLEANUP: config: slowstart is never negative

No need to check for a negative value in the "slowstart" argument, it's
an unsigned.

Reported-by: Dinko Korunic <dkorunic@reflected.net>
(cherry picked from commit 3a3bbcd6f1fd3b5629ca1759104b0e58ad637dc0)
---
 src/cfgparse.c |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/src/cfgparse.c b/src/cfgparse.c
index 90fdbff..e55d30a 100644
--- a/src/cfgparse.c
+++ b/src/cfgparse.c
@@ -3669,12 +3669,6 @@ stats_error_parsing:
 					err_code |= ERR_ALERT | ERR_FATAL;
 					goto out;
 				}
-				if (val < 0) {
-					Alert("parsing [%s:%d]: invalid value %d for argument '%s' of server %s.\n",
-					      file, linenum, val, args[cur_arg], newsrv->id);
-					err_code |= ERR_ALERT | ERR_FATAL;
-					goto out;
-				}
 				newsrv->slowstart = (val + 999) / 1000;
 				cur_arg += 2;
 			}
-- 
1.7.1

